dark/entrez.py

```diff
@@ -1,5 +1,6 @@
 from Bio import Entrez, SeqIO
-from urllib.error import URLError
+# from urllib.error import URLError
```

Comment:
Ah, sorry. I was testing to see if it worked and forgot to go back and delete. I should have read the diff more carefully.